command/certificate/create.go

```diff
@@ -217,6 +241,18 @@ func createAction(ctx *cli.Context) error {
 		return errs.EqualArguments(ctx, "CRT_FILE", "KEY_FILE")
 	}
 
+	notBefore, ok := flags.ParseTimeOrDuration(ctx.String("not-before"))
+	if !ok {
+		return errs.InvalidFlagValue(ctx, "not-before", ctx.String("not-before"), "")
+	}
+	notAfter, ok := flags.ParseTimeOrDuration(ctx.String("not-after"))
+	if !ok {
+		return errs.InvalidFlagValue(ctx, "not-after", ctx.String("not-after"), "")
+	}
+	if !notAfter.IsZero() && !notBefore.IsZero() && notBefore.After(notAfter) {
```

Comment:
Only thing I'm not sure about is whether this check wouldn't be better performed in the WithNotBeforeAfterDuration function.

Reply:
The check is there because I can return a proper error, it does not make sense to return a flag related error in WithNotBeforeAfterDuration.